content/document.ts

```diff
@@ -355,7 +355,7 @@ export const read = memoize((folderOrFilePath: string, ...roots: string[]) => {
   };
 });
 
-export function update(url: string, rawBody: string, metadata) {
+export async function update(url: string, rawBody: string, metadata) {
```

Comment:
Also needs to be `await`ed here:
https://github.com/mdn/yari/blob/5a9af2b3781562425ca57149a76cb0eb8563d457/content/document.ts#L576